src/docs/transactions.dox

```diff
@@ -246,6 +246,13 @@ For example, for the timestamp values <code>prepare_timestamp=100,
     timestamp is rounded up to the new prepare timestamp as
     <code>commit_timestamp=200</code>.
 
+Configuring <code>roundup_timestamps=(prepared=true)</code> also allows to set
+prepared timestamps smaller than or equal to the latest active read timestamps.
+Use this feature carefully as it may break repeated read. For example, if read
```

Comment:
The "For example..." comment is not a complete sentence. Also in the next sentence "putting a prepared update" is not really an action a user takes. The user only sets a prepared timestamp on a transaction.

Perhaps something like this to illustrate breaking repeatable read:
For example, consider a transaction with a read timestamp set to 30 and a key that has a value with timestamp 20. Before a prepared transaction (in another thread), reading the key returns the value. Now a prepared transaction elsewhere in the system sets a prepared timestamp at 30. The reader, trying to do a repeat read of the key, gets return a prepared conflict error instead of the value.